app/views/TaxFreeAmountView.scala.html

```diff
@@ -21,111 +21,122 @@
     ptaHeader(
-        displayBackLink = true,
+        displayBackLink = false,
         mainHeadingText = viewModel.header,
+        showGdsVersion = true,
         preHeadingText = Messages("tai.taxCode.preHeader"))
 }
 
-@mainTemplate(
+@main(
     title = viewModel.title,
     authedUser = Some(user),
-    pageHeader = Some(header),
-    articleClasses = Some("full-width")){
+    pageTitle = Some(Messages("tai.service.navTitle")),
+    backLinkUrl = Some(routes.TaxAccountSummaryController.onPageLoad.url),
+    backLinkContent = Messages("tai.back-link.upper"),
+    backLinkId = "backLink",
+){
 
-    <div class="grid-row">
+    <div class="govuk-grid-row">
 
-        <div class="grid grid-2-3">
-            <div class="inner-block">
+        <div class="govuk-grid-column-two-thirds">
+            @header
+        </div>
 
-                <div id="taxFreeAmountSummary" class="section flush--bottom">
+        <div class="govuk-grid-column-two-thirds">
 
-                    <p id="ledeMessage" class="flush--top">
-                        <span class="lede">@Messages("tai.taxFreeAmount.summarysection.taxFreeExplanation")</span>
+            <div id="taxFreeAmountSummary">
+                    <p class="govuk-body-l">
+                        @Messages("tai.taxFreeAmount.summarysection.taxFreeExplanation")
                     </p>
 
-                    <h2 class="flush--bottom">@Messages("tai.taxFreeAmount.summarysection.heading")
-                        <span class="display-block bold-large section section--narrow flush--bottom">@viewModel.annualTaxFreeAmount</span>
+                    <h2 class="govuk-heading-m">
+                        @Messages("tai.taxFreeAmount.summarysection.heading")
                     </h2>
 
-                    <p>@Messages("tai.taxFreeAmount.summarysection.p2")</p>
-                    <ul class="bullets">
+                    <p class="govuk-body govuk-!-font-size-36 govuk-!-font-weight-bold faded-text-dark">
+                        @viewModel.annualTaxFreeAmount
+                    </p>
+
+                    <p class="govuk-body">@Messages("tai.taxFreeAmount.summarysection.p2")</p>
+                    <ul class="govuk-list govuk-list--bullet">
                         <li>@Messages("tai.taxFreeAmount.summarysection.bullet1")</li>
                         <li>@Messages("tai.taxFreeAmount.summarysection.bullet2")</li>
                     </ul>
-                </div>
+            </div>
+            <div id="taxFreeAmountDetail" class="govuk-!-margin-top-8">
+                    <h2 class="govuk-heading-m">@Messages("tai.taxFreeAmount.detailsection.heading")</h2>
 
-                <div id="taxFreeAmountDetail" class="section">
-                    <h2 class="flush--bottom">@Messages("tai.taxFreeAmount.detailsection.heading")</h2>
 
                     @taxFreeAmountSummary(viewModel.taxFreeAmountSummary.summaryItems)
 
-                </div>
+            </div>
+                    <br>
```

Comment:
@masahmad are we allowed to use `<br>`

Reply:
good spot - yes we should take out the BR and use a bottom margin instead   - the accessibility team don't like anyone using BR's

Reply:
does it still look ok without the br